documentation/airport-format.md

```diff
@@ -1,309 +1,469 @@
+- **wind** ― The true heading (angle) in degrees and speed in knots of the current wind at the airport:
+```javascript
+"wind": {
+    "angle": 220,
+    "speed": 6
+}
+```
+
+
+### airspace
+ ```javascript
+ "airspace": [
+     {
+         "floor": 0,
+         "ceiling": 190,
+         "airspace_class": "B",
+         "poly": [
+             ["N35d57m50.000", "W115d51m15.000"],
+             ["N35d34m30.000", "W115d29m00.000"]
+         ]
+     }
+ ],
+```
+Position definition of the airport airspace.  Multiple airspace areas may be defined and will all be included in the airspace. This allows for advanced airspace stratification.
+- **_At least one airspace definition is required for an airport_**
+- **_All object properties are always required_**
+
+- **floor** ― The altitude (in feet) at which the airspace begins.
+- **ceiling** ― The altitude (in [FL][#flight-level] feet) at which the airspace ends.
+- **airspace_class** ― The FAA class of the airspace. _see [this FAA document](faa-airspace) for more details_
+- **poly** ― The coordinates of the airspace. in latitude, longitude: _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_
+
+
+### fixes
+```javascript
+"fixes": {
+    "_NAPSE068": ["N36.11211", "W115.14661"],
+    "BAKRR": ["N36.07582112978773", "W114.95309917207562"],
+    "BCE":   ["N37.68918661436860", "W112.30389943797489"],
+    "BESSY": ["N36.10772192196994", "W115.28956463349111"],
+    "BETHL": ["N36.88434886833625", "W112.44043432584908"],
+    "BIKKR": ["N36.56666216331978", "W116.75003219453492"]
+},
+```
+Each navaid located within or around the airport airspace in latitude, longitude: _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_.  Real life fixes are defined thusly:
+```javascript
+"BAKRR": ["N36.07582112978773", "W114.95309917207562"]
+```
+Sometimes, there is a need for a specific point in space that is not a real life fix.  In cases such as this, a fix can be created thusly:
+```javascript
+"_NAPSE068": ["N36.11211", "W115.14661"],
+```
+
+
+### runways
+```javascript
+"runways": [
     {
-      "name":        ["36", "18"],     // the name of each end of the runway
-      "name_offset": [[0, 0], [0, 0]], // the offset, in km, of the runway text when drawn on the map
-      "end":         [                 // the ends of the runway
-                       ["lat", "lon", "elevation"],
-                       ["lat", "lon", "elevation"]
-                     ],
-      "delay":       [2, 2],           // the number of seconds it takes to taxi to the end of the runway
-      "sepFromAdjacent": [2.5, 2.5],   // Distance in nautical miles that another aircraft can come while on parallel approach paths, a violation will occur at 85% of this value
-      "ils":         [true, false]     // not used yet; indicates whether or not that end of the runway has ILS
+        "name": ["07L", "25R"],
```

Comment:
It's a convention we should continue with actually... it's just so we treat each "runway" (line of tarmac on the earth) as a single object, and that it has different ends that are given names. So a "one-runway airport" has two runway names (from each end), but just one entry in `runways`, etc